build.gradle

```diff
@@ -224,6 +227,9 @@ subprojects {
     repositories {
       mavenLocal()
       mavenCentral()
+      maven {
```

Comment:
Can you not just use `jcenter()`?

Reply:
we dont publish avro-util to jcenter or central (yet?)

Reply:
Nvm, I forgot about the distinction between "central" and our own org repo on Bintray